packages/compass-aggregations/src/components/stage-toolbar/index.tsx

```diff
@@ -132,13 +183,32 @@ export function StageToolbar({
       </div>
       <div className={rightStyles}>
         {showFocusMode && (
-          <IconButton
-            onClick={() => onFocusModeEnableClick(index)}
-            aria-label="Open stage in focus mode"
-            data-testid="focus-mode-button"
-          >
-            <Icon glyph="FullScreenEnter" size="small"></Icon>
-          </IconButton>
+          <>
+            <GuideCue
+              data-testid="focus-mode-guide-cue"
+              open={isGuideCueIntersecting && isGuideCueVisible}
+              setOpen={() => setGuideCueVisited()}
+              refEl={focusModeButtonRef}
+              numberOfSteps={1}
+              popoverZIndex={2}
+              scrollContainer={containerRef.current}
+              portalContainer={containerRef.current}
+              title="Focus Mode"
+              tooltipAlign="bottom"
```

Comment:
The way our logic for showing the cue (in particular the scroll container options we use so that the cue can scroll with the ref el) combines with the way leafygreen popups lock focus and thus auto scrolls the parent to fit the focused content leads to a pretty nasty behavior where as soon as we render the cue the current scroll position is completely reset:

![Kapture 2023-02-09 at 18 59 40](https://user-images.githubusercontent.com/5036933/217898731-e8918c70-0212-4a58-8587-4d2f69190b1d.gif)

([here's the minimal repro](https://codesandbox.io/s/restless-architecture-8tqrn8?file=/src/App.js) that I made)

I am not really sure that we can do anything about it here, other than either removing all the container props (but this means that the cue will not scroll) or removing the `topAlign` here, at least in that case the chances of focusable elements inside the cue not being on the screen and stealing the scroll position will be minimized?